test/integration/install_test.go

```diff
@@ -462,7 +462,14 @@ func TestInstallOrUpgradeCli(t *testing.T) {
 		testutil.AnnotatedFatalf(t, "'kubectl apply' command failed",
 			"'kubectl apply' command failed\n%s", out)
 	}
-	expectedVizDeployments := testutil.LinkerdVizDeployReplicas
+
+	// It is necessary to clone LinkerdVizDeployReplicas so that we do not
+	// mutate it's original value.
+	vizDeployments := testutil.LinkerdVizDeployReplicas
+	expectedVizDeployments := make(map[string]testutil.DeploySpec)
+	for k, v := range vizDeployments {
```

Comment:
TIOLI: No need to introduce `vizDeployments` and instead can iterate directly over `testutil.LinkerdVizDeployReplicas`.